src/Symfony/Bundle/FeatureToggleBundle/FeatureToggleBundle.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FeatureToggleBundle;
+
+use Symfony\Bundle\FeatureToggleBundle\DependencyInjection\CompilerPass\DebugPass;
+use Symfony\Bundle\FeatureToggleBundle\DependencyInjection\CompilerPass\FeatureCollectionPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+
+final class FeatureToggleBundle extends Bundle
```

Comment:
having a new bundle does not match the PR description where you show it being configured as part of the FrameworkBundle configuration.

Reply:
Indeed. It was something I wanted to discuss in the PR actually. I first worked on this as a separate package a few years ago and I've reworked it lately. I was wondering if it should be merged onto FrameworkBundle or should it be a separate one ? Should I update the description while it is decided ?

Reply:
This should go in FrameworkBundle, like all core components except Security ones.